ReactViewControl/ReactViewRender.cs

```diff
@@ -34,6 +34,8 @@ internal partial class ReactViewRender : IChildViewHost, IDisposable {
         private ResourceUrl defaultStyleSheet;
         private bool isInputDisabled; // used primarly to control the intention to disable input (before the browser is ready)
 
+        private Lazy<EditCommands> editCommands;
```

Comment:
I know that I suggested that but now looking again using the Lazy is pointless... the overhead of instantiating the EditCommands is the same as instantiang the lazy... I'll remove the same pattern from webview